drizzlepac/haputils/product.py

```diff
@@ -64,7 +64,7 @@ class HAPProduct:
     """
 
     def __init__(
-        self, prop_id, obset_id, instrument, detector, filename, filetype, log_level
+        self, prop_id, obset_id, instrument, detector, aperture, filename, filetype, log_level
     ):
```

Comment:
As noted in our discussion, I would rename this variable to be self.scale_aperture (or some other name with the work aperture in it) indicating that while it is the "aperture", it is not necessarily the value of the APERTURE keyword in the RAW FITS headers.

Reply:
I will change all of the mentions of `aperture` to something less vague. How about `aperture_from_poller`?

Reply:
This is fine - very explicit.